test/Interop/Cxx/symbolic-imports/print-symbolic-module-interface.swift

```diff
@@ -60,6 +60,19 @@ public:
     };
 };
 
+#define IMMORTAL_FRT                                                         \
+    __attribute__((swift_attr("import_reference")))                              \
+    __attribute__((swift_attr("retain:immortal")))                               \
+    __attribute__((swift_attr("release:immortal")))
+
+struct IMMORTAL_FRT MyImmortal {
+    virtual void foo() const {};
+};
+
+struct NonCopyable {
+    NonCopyable(const NonCopyable& other) = delete;
+};
```

Comment:
Can we add a test with a private copy constructor? they should be imported as non-copyable right?

Reply:
And also a test with a non-default and explicitly defaulted move constructors?

Reply:
We have a test for these kinds of C++ types, `type-classification-module-interface.swift`. Right now we don't import structs with private copy constructors, but you're right, we should be importing them as non-copyable in Swift. I'll put up a separate patch for that.